### DISCUSSION TITLE
Disabling suggestions for text box entry

### DISCUSSION THREAD

**Discussion by veranicolette** [jspsych/jsPsych#1214] [Q&A] [answered]
Hello, is there a way I can disable the text suggestions that come up on a text box when using the survey-html-form plugin?

**Comment by becky-gilbert** [comment, marked as answer]
Yes, we added this to the jsPsych 6.2 release, so you can use [the updated survey-html-form plugin](https://github.com/jspsych/jsPsych/blob/master/plugins/jspsych-survey-html-form.js) which includes an `autocomplete` parameter. Or you can modify your plugin file by adding `autocomplete="off"` to the `<form>` element.
